modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/errors/Errors.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.console.errors;
+
+import javax.cache.CacheException;
+import org.apache.ignite.IgniteClientDisconnectedException;
+import org.apache.ignite.IgniteException;
+
+/**
+ * Class with error codes and messages.
+ */
+public class Errors {
```

Comment:
Why we need string constant? I can't find same pattern in spring library

Reply:
String constant better for refactoring cases, and we can find all usages in one click.

Reply:
In IDEA you can click on message code in properties file and receive all usage. If you need to change message code you should change const then change message code it's very strange.

Reply:
Well, let's consider the following case:
We change the message code in the properties file.

Without constants, we must change each file with a changed message code.
With constants, we have to change one place and not worry about missing changes in other files.

With or without constants, we have to change more than one place, and I think one change is better than a change in many files.